tests/testutils/src/main/kotlin/io/element/android/tests/testutils/IsInDebug.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.tests.testutils
+
+/**
+ * Returns true if the app is in debug mode.
+ */
+fun isInDebug() = BuildConfig.DEBUG
```

Comment:
This is what I call "API renaming", I used to do quite a good amount of this in another codebase and then started wondering if I really should:
- On one side I like it because it make things more idiomatic.
- On the other it makes the code less readable to new comers (every Android dev knows what `BuildConfig.DEBUG` is but nobody knows what `isInDebug()` so it could potentially do anything).

Reply:
I just did this because `BuildConfig.DEBUG` didn't seem to be accessible in unit tests. Maybe by generating resources it would work?

Reply:
`BuildConfig` generation is optional since AGP8, maybe that's the issue?
But even better question, why do you need BuildConfig in a unit test, what are you trying to do?

Reply:
No, the `BuildConfig` generation is already enabled for that module. As for why I need this, it's because `UserId("foo")` will crash in debug mode so we can track invalid ids but will work fine in release mode. Since we have some code that needs to check for invalid ids in release mode in these classes we need to take into account both cases.

Another option would be to remove this check.